scss/_variables.scss

```diff
@@ -636,6 +635,7 @@ $form-range-thumb-focus-box-shadow:        0 0 0 1px $body-bg, $input-focus-box-
 $form-range-thumb-focus-box-shadow-width:  $input-focus-width !default; // For focus box shadow issue in IE/Edge
 $form-range-thumb-active-bg:               lighten($component-active-bg, 35%) !default;
 $form-range-thumb-disabled-bg:             $gray-500 !default;
+$form-range-thumb-transition:               background-color .15s ease-in-out, border-color .15s ease-in-out, box-shadow .15s ease-in-out !default;
```

Comment:
Should we make this available to all other forms like checks/radios?

Reply:
Yep! I was aware of that when I saw the TODO comment in the `_form-check.scss`, but I prioritize removing "custom" in this PR.